mix.exs

```diff
@@ -13,7 +13,7 @@ defmodule Wallaby.Mixfile do
   def project do
     [app: :wallaby,
      version: @version,
-     elixir: "~> 1.7",
+     elixir: "~> 1.5",
```

Comment:
I'd highly suggest to add all now supported elixir versions (minor not all patch levels) back to the travis matrix so that we make sure that we don't break compatibility.

Reply:
Makes sense, on it!